### PR TITLE
Nerfs Abductors

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Abductor baton attack delay increased from 0 seconds to 1.5 seconds.
Abductor baton stun time reduced from 14 seconds to 10 seconds.
Abductor baton time to cuff increased from 3 seconds to 5 seconds.
Abductor stealth module now has a cooldown which is shared with the combat cooldown.
Abductor combat module now gives an adrenaline chemical similar to changeling adrenaline.
Abductors now require air to breathe.
Abductors can get viruses.
The following glands were removed:
- Plasma
- Slime
- Spider
- Species Transformation
- Viral

## Why It's Good For The Game

Abductors are currently stupidly strong and are currently not balanced around escalation rules as well as all the security reworks. This PR nerfs them  so they can no longer fight an entire team of security 2 v 8 style and win with zero contest, regardless of skill level.

This PR effectively makes it so you can still gank crew 1v1 or even 1v2, but you start to lose at around 1v3. It also makes abductors susceptible to viruses (gotta be sanitary now and avoid blood/killing) and are now required to breathe.

## Changelog
:cl: BurgerBB
balance: Nerfs abductor baton. Stealth module now has a cooldown that's shared with the combat module. Abductors are now required to breathe and can get viruses.
/:cl:

### CODE REVIEW

**Comment by BurgerLUA** [comment]
-8 more like negative ate my ass.

**Comment by Vizoplez** [comment]
You have literally zero clue why abudctors have this stuff, so let me explain

-Abductors have a gland that causes random, airborne viruses
-Them getting viruses, cripples their ability to function as an antagonist

this alone gets the PR denied on principle.
other than that change good

File: modular_skyrat/master_files/code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//Used for Abductors
+/datum/reagent/medicine/abductoradrenaline
+	name = "Z-Gamma Precepts"
+	description = "A synthesized adrenaline based on a chemical a certain other lifeform creates. Overdosing can cause brain damage."
+	color = "#4CFF00"
+	overdose_threshold = 30
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/medicine/changelingadrenaline/on_mob_life(mob/living/carbon/metabolizer, delta_time, times_fired)
```
Comment (Seris02) [inline modular_skyrat/master_files/code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: not sure if you meant for this to be changelingadrenaline or abductoradrenaline, just a heads up

**Comment by BurgerLUA** [comment]
> You have literally zero clue why abudctors have this stuff, so let me explain
> 
> -Abductors have a gland that causes random, airborne viruses
> -Them getting viruses, cripples their ability to function as an antagonist
> 
> this alone gets the PR denied on principle.
> other than that change good

sounds like the gland is shit then

File: modular_skyrat/master_files/code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (BurgerLUA) [reply modular_skyrat/master_files/code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: fixed

**Comment by Gandalf2k15** [comment]
Do this upstream. I really don't want us fucking with balance regarding antags. It always ruins said antag. So allow TG to analyse the changes and take it from there.

**Comment by BurgerLUA** [comment]
Skyrat balance is entirely different from /tg/ balance, though.

**Pull request closed without merging**